src/pluginplay/module_manager.cpp

```diff
@@ -79,4 +79,7 @@ ModuleManager::runtime_type& ModuleManager::get_runtime() const noexcept {
     return pimpl_->get_runtime();
 };
 
+std::vector<type::key> ModuleManager::keys() const noexcept {
```

Comment:
typedef and nothrow comment apply here too.

Reply:
Done.